lib/Runtime/ByteCode/OpCodes.h

```diff
@@ -381,6 +381,7 @@ MACRO_WMS_PROFILED(     LdSlot,                     ElementSlot,    OpTempNumber
 MACRO_WMS_PROFILED(     LdEnvSlot,                  ElementSlotI2,  OpTempNumberSources)
 MACRO_WMS_PROFILED(     LdInnerSlot,                ElementSlotI2,  OpTempNumberSources)
 MACRO_WMS_PROFILED(     LdLocalSlot,                ElementSlotI1,  OpTempNumberSources)
+MACRO_EXTEND_WMS(       LdParamSlot,                ElementSlotI1,  OpTempNumberSources)
```

Comment:
Cool. Is the Profiled opcode now being emitted with your changes, or do we also need a ByteCodeWriter change to make that happen (and an interpreter change to consume it)?